cg/models/pipeline_qc_metrics/metrics_yaml_fields.py

```diff
@@ -0,0 +1,17 @@
+"""This module holds the definition of the field present in the pipeline qc metrics yaml file."""
+from pydantic import BaseModel
+
+
+class PipelineQCMetricsFields(BaseModel):
+    """Definition for elements in deliverables metrics file."""
+
+    header: str  # describes the level of the metric sample or case
+    id: str  # sample id
```

Comment:
Could also be case level - right?

Reply:
Yes, that could work. But I don't think that is will be necessary. A pipelineqcmetric document will ever be only for one case at a time but can have multiple different samples. so it might be less confusing only providing a sample id here for different samples